cmd/schema_gen.go

```diff
@@ -41,13 +42,25 @@ var schemaGenCmd = &cobra.Command{
 
 		// Build a graph from the given filter.
 		fileSystem, _ := compileAllModels()
+
 		graph := buildGraph(fileSystem, ModelFilters)
 
 		// Generate schema for every file in the graph concurrently.
 		if err := generateSchemaForGraph(graph); err != nil {
 			fmt.Printf("❌ %s\n", err)
 			os.Exit(1)
 		}
+
+		// refresh the graph state for doc suggestions
+		fmt.Println("\n🌀 Resetting graph run status for doc string suggestions")
```

Comment:
Nit; I'm not sure this line is really needed as the process bars will show us this happening